website/source/docs/enterprise/index.html.md

```diff
@@ -29,7 +32,21 @@ Enterprise](https://www.hashicorp.com/consul.html).
 Licensing capabilities were added to Consul Enterprise v1.1.0. The license is set
 once for a datacenter and will automatically propagate to all nodes within the
 datacenter over a period of time scaled between 1 and 20 minutes depending on the
-number of nodes in the datacenter. The license can be set via the 
+number of nodes in the datacenter. There are two methods for licensing Consul
+enterprise.
+
+### Included in the Enterprise Package
+
+If you are downloading Consul from Amazon S3, then the license is included
```

Comment:
The temporary license binaries are available in S3 too. Up until the last month or so that was the only place any of the binaries existed.

The zip packages containing `+pro` or `+prem` in the name in S3 are the ones with builtin licenses. The zip packages that have `+ent` in the name are the same ones that we now copy to releases.hashicorp.com

Reply:
@mkeeler Thanks for the info! I've update the page, lmk if it needs additional clarification.